### ISSUE TITLE
False Negative - Isset operations variables existence

### ISSUE THREAD

**Issue by funivan** [kalessil/phpinspectionsea#666]
True positive
```php
function test(){
  echo $result ?? 1; // <-- All is ok: $result is marked as variable that is not initialized
}
```
False negative
```php
function test(){
  echo $result[1] ?? 1; // <-- Inspection error: $result is not marked
}
```

I think inspection can be enhanced and check if the variable exists. 
p.s. Maybe this can be part of ultimate edition?

**Comment by kalessil** [comment]
Done.
